src/FsLibLog/FsLibLog.fs

```diff
@@ -723,9 +790,11 @@ module LogProvider =
     /// **Output Type**
     ///   * `ILog`
     ///
-    let getLoggerFor<'a> () =
+    let inline getLoggerFor<'a> () =
```

Comment:
This probably doesn't change anything but any reason it was made `inline`?

Reply:
I thought I had gotten an error message about generics erasure and using `inline` resolved it. Futher testing shows that, yes, `inline` resolves an error, but the function itself doesn't work in Fable - `GetCurrentMethod()` returns `null`. I'll move this inside a compiler flag, too. This raises some concerns about the fact that the only way to get a logger is by name or type.

Reply:
Interesting.  Might be worth bring up to the fable team too but i'll mark this as resolved.